ui/job-view/headerbars/FiltersMenu.jsx

```diff
@@ -56,7 +56,7 @@ function FiltersMenu(props) {
             {resultStatusMenuItems.map(filterName => (
               <span key={filterName}>
                 <span>
-                  <label className="dropdown-item">
+                  <label htmlFor={filterName} className="dropdown-item">
```

Comment:
Same as earlier re redundant label for/wrapping element. However, in this case, we probably don't need the attribute, since it's already wrapping.

Reply:
The linter was giving me grief on this without doing both wrapping and the ``htmlFor``.  However, switching to the ``reactstrap`` (react bootstrap) component of ``Label`` works much better, so moving to that.